wp-content/themes/inf_theme/theme/class-theme.php

```diff
@@ -17,36 +16,15 @@
  */
 class Theme extends Config {
 
-  /**
-   * General Helper class
-   *
-   * @var object General_Helper
-   *
-   * @since 2.0.1
-   */
-  public $general_helper;
-
-  /**
-   * Initialize class
-   *
-   * @param Helpers\General_Helper $general_helper Helper class instance.
-   *
-   * @since 3.0.0 Removing constructor and global variables.
-   * @since 2.0.0
-   */
-  public function __construct( General_Helper $general_helper ) {
-    $this->general_helper = $general_helper;
-  }
-
   /**
    * Register the Stylesheets for the theme area.
    *
-   * @since 2.0.0
+   * @since 1.0.0
    */
   public function enqueue_styles() {
 
-    $main_style = $this->general_helper->get_manifest_assets_data( 'application.css' );
-    wp_register_style( static::THEME_NAME . '-style', $main_style );
+    $main_style = General_Helper::get_manifest_assets_data( 'application.css' );
+    wp_register_style( static::THEME_NAME . '-style', $main_style, array(), static::THEME_VERSION );
```

Comment:
Same comment as above about versions and cache busting.

Reply:
answered ⬆️